video/out/vo_vaapi_wayland.c

```diff
@@ -33,17 +33,17 @@
 struct va_pool_entry {
     /* key */
     VASurfaceID surface;
-
     VADRMPRIMESurfaceDescriptor desc;
     struct wl_buffer *buffer;
     struct zwp_linux_buffer_params_v1 *params;
-    uint drm_format;
+    uint32_t drm_format;
 };
+
 struct va_pool {
     struct vo *vo;
     struct va_pool_entry **entries;
-    uint num_entries;
-    uint num_allocated;
+    unsigned int num_entries;
+    unsigned int num_allocated;
```

Comment:
by the way: `int` is also commonly used for array sizes and such (even if it can never be negative)

Reply:
I prefer to use unsigned types for values that will never be negative